app/controllers/providers/address_selections_controller.rb

```diff
@@ -20,8 +20,18 @@ def update
       render :show unless save_continue_or_draft(@form)
     end
 
+    def add_a_space_to_postcode
+      string = @form.postcode.call(insert(-3), ' ')
+      string
+    end
+    helper_method :add_a_space_to_postcode
+
     private
 
+    # def add_a_space_to_postcode(string)
+    #   string.(insert(-3), " ")
+    # end
+
```

Comment:
Remove these comments?

Reply:
apologies, I had not pushed my most recent changes, these are removed